build.gradle

```diff
@@ -75,6 +75,15 @@ task dexify(type: Jar){
 
 task buildDex dependsOn "build", "dexify"
 
+task buildMove(dependsOn: build){
+    doLast{
+        copy{
+            from "build/libs/UnitInfo.jar"
```

Comment:
~~Unit Info? Wrong mod.~~

Reply:
LMAO

Reply:
Lmfao i made a pr to unitinfo that does the same thing then forgot to change it

Reply:
forgor

Reply:
h

Reply:
forgorh